src/InterprojectExchange/InterprojectExchangeSaver.cs

```diff
@@ -280,35 +288,27 @@ private void AddOrUpdateEplannerVersionInfo(IProjectModel model)
             var res = string.Empty;
 
             string ipComment = "-- адрес удаленного контроллера";
-            string ipEmulatorComment = "-- адрес удаленного контроллера при " +
-                "эмуляции на столе";
+            string ipEmulatorComment = "-- адрес удаленного контроллера при эмуляции на столе";
             string emulationComment = "-- включение эмуляции";
-            string cycleTimeComment = "-- время ожидания между опросами " +
-                "контроллера";
+            string cycleTimeComment = "-- время ожидания между опросами контроллера";
             string timeoutComment = "-- таймаут для modbus клиента";
             string portComment = "-- modbus - порт удаленного контроллера";
             string enabledComment = "-- включить/выключить шлюз";
-            string stationComment = "-- номер станции modbus удаленного " +
-                "клиента";
-
-            res += $"\t[\'{projectName}\'] =\n\t{{\n";
-            res += prefix + $"ip = \'{pacInfo.IP}\',\t{ipComment}\n";
-            res += prefix + $"ipemulator = \'{pacInfo.IPEmulator}\',\t" +
-                $"{ipEmulatorComment}\n";
-            res += prefix + $"emulation = " +
-                $"{pacInfo.EmulationEnabled.ToString().ToLower()},\t" +
-                $"{emulationComment}\n";
-            res += prefix + $"cycletime = {pacInfo.CycleTime},\t" +
-                $"{cycleTimeComment}\n";
-            res += prefix + $"timeout = {pacInfo.TimeOut},\t{timeoutComment}\n";
-            res += prefix + $"port = {pacInfo.Port},\t{portComment}\n";
-            res += prefix + $"enabled = " +
-                $"{pacInfo.GateEnabled.ToString().ToLower()},\t" +
-                $"{enabledComment}\n";
-            res += prefix + $"station = {pacInfo.Station},\t" +
-                $"{stationComment}\n";
-
-            return res;
+            string stationComment = "-- номер станции modbus удаленного клиента";
+
+            const int margin = -17;
+
+            return new StringBuilder()
+                .Append($"\t['{projectName}'] =\n\t{{\n")
+                .Append($"{prefix}ip         = {$"'{pacInfo.IP}',", margin} {ipComment}\n")
+                .Append($"{prefix}ipemulator = {$"'{pacInfo.IPEmulator}',", margin} {ipEmulatorComment}\n")
+                .Append($"{prefix}emulation  = {$"{pacInfo.EmulationEnabled.ToString().ToLower()},", margin} {emulationComment}\n")
+                .Append($"{prefix}cycletime  = {$"{pacInfo.CycleTime},", margin} {cycleTimeComment}\n")
+                .Append($"{prefix}timeout    = {$"{pacInfo.TimeOut},", margin} {timeoutComment}\n")
+                .Append($"{prefix}port       = {$"{pacInfo.Port},", margin} {portComment}\n")
+                .Append($"{prefix}enabled    = {$"{pacInfo.GateEnabled.ToString().ToLower()},", margin} {enabledComment}\n")
+                .Append($"{prefix}station    = {$"{pacInfo.Station},", margin} {stationComment}\n")
+                .ToString();
```

Comment:
Для улучшения читабельности предлагаю добавить переменные:
```suggestion
            string EmulationEnabled = pacInfo.EmulationEnabled.ToString().ToLower();
            string GateEnabled = pacInfo.GateEnabled.ToString().ToLower();
            
            return new StringBuilder()
                .Append($"\t['{projectName}'] =\n\t{{\n")
                .Append($"{prefix}ip         = {$"'{pacInfo.IP}',", margin} {ipComment}\n")
                .Append($"{prefix}ipemulator = {$"'{pacInfo.IPEmulator}',", margin} {ipEmulatorComment}\n")
                .Append($"{prefix}emulation  = {$"{EmulationEnabled },", margin} {emulationComment}\n")
                .Append($"{prefix}cycletime  = {$"{pacInfo.CycleTime},", margin} {cycleTimeComment}\n")
                .Append($"{prefix}timeout    = {$"{pacInfo.TimeOut},", margin} {timeoutComment}\n")
                .Append($"{prefix}port       = {$"{pacInfo.Port},", margin} {portComment}\n")
                .Append($"{prefix}enabled    = {$"{GateEnabled},", margin} {enabledComment}\n")
                .Append($"{prefix}station    = {$"{pacInfo.Station},", margin} {stationComment}\n")
                .ToString();
```